shared/templates/file_permissions/oval.template

```diff
@@ -1,3 +1,19 @@
+{{%- if ALLOW_STRICTER_PERMISSIONS %}}
+    {{#
+        When allowing stricter permissions, the check looks for files/directories that
+        do not match the permissions (STATEMODE). The criterion is then negated to make
+        sure there are no files/directories matched.
+    #}}
+  {{% set FILE_EXISTENCE = "at_least_one_exists" %}}
```

Comment:
I think this should be `any_exist`.
If there are no files/directories with extra permissions bits set, we are fine.

Reply:
If we use `any_exist` every check would fail. For example, if no object is collected (there are no files/directories with wrong permissions), this check would pass (`any_exist` allows) and subsequentially fail because the criterion has `negate=true`. That means, we should keep the `at_least_one_exists` to cover this case.

Reply:
You are right, the `negate` confused me, :)